scipy/integrate/__init__.py

```diff
@@ -17,12 +17,9 @@
    tplquad       -- General purpose triple integration
    nquad         -- General purpose N-D integration
    fixed_quad    -- Integrate func(x) using Gaussian quadrature of order n
-   quadrature    -- Integrate with given tolerance using Gaussian quadrature
-   romberg       -- Integrate func using Romberg integration
    newton_cotes  -- Weights and error coefficient for Newton-Cotes integration
    qmc_quad      -- N-D integration using Quasi-Monte Carlo quadrature
    IntegrationWarning -- Warning on issues during integration
-   AccuracyWarning  -- Warning on issues during quadrature integration
```

Comment:
Technically note deprecated however only emitted by the two functions being removed